README.md

```diff
@@ -18,15 +18,20 @@ Download the extension from [Visual Studio Marketplace](https://marketplace.visu
 
 ## Versioning
 
-The language service extension uses the `@angular/language-service` package for its backend. This
-package is loaded either from the version bundled with the extension, or from the current workspace
-project the extension is running on. Due to the behavior of TypeScript below version 3.8,
-incompatible versions of the language service may sometimes be loaded. If you are using a version of
-TypeScript below 3.8, we suggest either
-
-- Not installing `@angular/language-service` in your project (recommended; will fallback on the
-    version bundled with the extension)
-- Installing and keeping updates for the latest version of `@angular/language-service`
+The language service extension relies on the `@angular/language-service` and `typescript` packages
+for its backend. These packages loaded, in order of priority, from:
+
+1. The path specified by `angular.ngdk` and `typescript.tsdk`, respectively, in project or global
+   settings.
+2. The version of these packages bundled with the Angular Language Service extension.
+3. The version of these packages present in the current workspace's node_modules.
+
+If the `@angular/language-service` and `typescript` packages are loaded by methods (1) or (3), there
+is a potential for a mismatch between the API expected by `@angular/language-service` and the API
+provided by `typescript`. This could lead to a failure of the language service extension.
+
+For this reason, we suggest not specifying either `angular.ngdk` nor `typescript.tsdk` in your
```

Comment:
Could you please move this warning to the second paragraph, before the explanation? Like

> We suggest **not** using method (1) or (3) because there is a potential for mismatch ...